src/protocol/modulus_conversion/convert_shares.rs

```diff
@@ -24,7 +23,6 @@ pub struct ConvertShares {
 #[derive(Debug, Clone, Copy, PartialEq, Eq, Hash)]
 enum Step {
```

Comment:
Do we still need this?

Reply:
I kill it in the next diff in this stack: #233